src/main/java/net/bramp/ffmpeg/FFmpegUtils.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Helper class with commonly used methods
  */
-public final class FFmpegUtils {
+public abstract class FFmpegUtils {
```

Comment:
I would expect anyone to extend this class (which is what abstract implies). Instead I used a final class, with private constructor. http://stackoverflow.com/questions/25223553/how-can-i-create-an-utility-class